content/src/fieldConfig/config.json

```diff
@@ -50,6 +50,7 @@
   "navigationDefaults": {
     "navBarUnnamedOwnedBusinessText": "Unnamed Business",
     "navBarUnnamedForeignRemoteSellerWorkerText": "Unnamed Out-of-State Business",
+    "navBarUnnamedDbaBusinessText": "Unnamed DBA",
```

Comment:
[sand] I know this was already decided, but I think it was decided kind of quickly without a lot of thought and should be confirmed (maybe a slack to the content channel? I can do it if you want)

I don't think this should say "Unnamed DBA" because it's grammatically incorrect and does not make sense. A DBA is not a type of business, like Nexus or something. It's not a noun, it's an adjective.

DBA stands for "Doing Business As" and is an adjective that describes a name - you **HAVE** a "DBA Name", you cannot **BE** "a DBA".

I think this content needs to be revised.

Reply:
I can do it